Automattic/jetpack
# VideoPress: Fix JITM layout on video edit page

Fixes #30408

## Proposed changes:

* Adds a container on video edit page with specific CSS
* Adds `jetpack-videopress` to the list of pages to refetch the JITM on hash change

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Have a JITM showing up on the VideoPress page. One way to do this is to have an unattached Jetpack Complete plan.
* Go to the VideoPress dashboard
* Check that the JITM displays as normal
* Navigate to a video's details edit page
* Check that the JITM is displayed in that page as well and the layout is not broken
* Click on `Go back` and check that the JITM is re-applied to the dashboard

https://user-images.githubusercontent.com/8486249/236276826-7d6f0f9b-2418-4944-b449-00be9e9a3a9b.mp4
